uaa/remove-user-guid-from-all-groups.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+usage() {
+  echo "Usage: "
+  echo "  $0 <user-guid>"
+  echo " "
+  echo "  <user-guid>: The guid of the user to remove from all groups."
+  echo " "
+}
+
+main() {
+  if [[ ! $# -eq 1 ]]; then
+    usage
+    exit 1
+  fi
+  local user_guid=$1
+  local group_guids=$(uaac curl /Groups | awk '/BODY/{y=1;next}y' | jq -r '.resources[] | select(.members[] | select(.value | contains("'"${user_guid}"'"))) | .id')
```

Comment:
can we check if the value _is_ `user_guid`, rather than if it contains `user_guid`?

Reply:
not based on the structure. `members` is an array with multiple objects that have the `value` property.